var/spack/repos/builtin/packages/py-breathe/package.py

```diff
@@ -38,6 +39,7 @@ class PyBreathe(PythonPackage):
     depends_on("py-sphinx", type=("build", "run"))
     depends_on("py-sphinx@:3", type=("build", "run"), when="@:4.20")
     depends_on("py-sphinx@3:3.2", type=("build", "run"), when="@4.21:4.32")
-    depends_on("py-sphinx@3:", type=("build", "run"), when="@4.33:")
+    depends_on("py-sphinx@3:4", type=("build", "run"), when="@4.33:")
+    depends_on("py-sphinx@4.0:4,5.0.1:", type=("build", "run"), when="@4.34:")
```

Comment:
```suggestion
    depends_on("py-sphinx@4.0:4,5.0.1:5", type=("build", "run"), when="@4.34:")
```
Sphinx 6 is not yet supported.